### PR TITLE
Coupon UI fixes

### PR DESCRIPTION
**Pull request by dsego**
Depends on #476
- collection url is again /api/v2/coupons/
- remove "Your Platform Name Here" from footer (SOL-1540)
- ~~default value for category~~

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @dsego! It looks like you're a member of a company that does contract work for edX. If you're doing this work as part of a paid contract with edX, you should talk to edX about who will review this pull request. If this work is not part of a paid contract with edX, then you should ensure that there is an OSPR issue to track this work in [JIRA](https://openedx.atlassian.net), so that we don't lose track of your pull request.

To automatically create an OSPR issue for this pull request, just visit this link: https://openedx-webhooks.herokuapp.com/github/process_pr?repo=edx%2Fecommerce&number=486

**Comment by dsego** [comment]
@vkaracic @marjev Please take a look.

**Comment by marjev** [comment]
:+1:  lgtm

File: ecommerce/static/js/models/coupon_model.js [hunk 1]
```diff
@@ -31,7 +31,8 @@ define([
             defaults: {
                 quantity: 1,
                 stock_record_ids: [],
-                code: ''
+                code: '',
+                category: ''
```
Comment (vkaracic) [inline ecommerce/static/js/models/coupon_model.js]: The coupon category feature is currently on hold. You don't need to send category for now.

**Comment by vkaracic** [comment]
Apart from the category code, LGTM :+1:

**Comment by mjfrey** [comment]
good.

:+1:  Please squash

**Pull request merged**